### ISSUE TITLE
[Command] Missing options

### ISSUE THREAD

**Issue by fsevestre** [phpstan/phpstan-symfony#50]
Hello,

I am adding the `console_application_loader` config part on my PHPStan configuration and I'm facing an issue with Symfony's defaults options not being found.

`Command "..." does not define option "env".`

We are using the `env` options to hide some dynamic informations on test environment for our test suite.

My `console-application.php` file
```php
<?php

use Symfony\Bundle\FrameworkBundle\Console\Application;

require __DIR__.'/../vendor/autoload.php';

$kernel = new AppKernel('test', false);

return new Application($kernel);

```

-----

I tried to create a patch, but without success. I don't find a way to retrieve those default options in the `UndefinedOptionRule` class.

**Comment by lookyman** [comment]
Ah, yes, that might be a bug. I'll have a look at it over the weekend. Thanks.

**Comment by lookyman** [comment]
Fixed in 1c995438481cd49e5724d620f7eb57cca7c1b2ca

**Comment by benr77** [comment]
I'm still experiencing this bug in 0.11.6
